docs/pages/desktop-access/directory-sharing.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Desktop Access Directory Sharing
+description: Teleport Desktop Access Directory Sharing lets you easily send files to a remote desktop.
+---
+
+Directory Sharing is a feature of Teleport Desktop Access that makes it easy to
+move files between a local machine and a remote desktop—and apply changes to
+those files—without compromising security.
+
+During a remote desktop session, you can select a folder on your local
+workstation to share with the remote desktop. Changes to the folder on either
+the remote desktop or your workstation are reflected on both machines for the
+duration of the session. 
+
+The shared directory makes it convenient to download log files, edit
+configuration files, or perform any other file manipulations on a remote Windows
+desktop. You can disable Directory Sharing for specific users via their Teleport
+roles, and use session recording to audit activity in the shared directory
+after the session ends.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Teleport Desktop Access with at least one remote desktop registered in your
+  cluster. If you have not yet configured Desktop Access, read [Getting Started
+  with Desktop Access](./getting-started.mdx) before beginning this guide.
+- A browser on your local machine that supports the File System Access API,
+  which Teleport uses for Directory Sharing. We support the latest versions of
+  Chromium-based browsers like Google Chrome, Brave, and Microsoft Edge. You can
+  see a full compatibility table in the [Mozilla Developer Network
+  documentation](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API#browser_compatibility).
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Share a directory
+
+To share a directory, open the Teleport Web UI and begin a desktop session by
+navigating to the "Desktops" tab on the sidebar, finding a desktop you want to
+connect to, and clicking "CONNECT".
+
+Once the session starts, click the three-dot menu on the top-right of the screen
+and click "Share Directory":
+
+![The "Share Desktop" button](../../img/desktop-access/share.png)
+
+When you first share a directory, your browser will prompt you to allow it to
+make changes in the directory, and you will need to grant these permissions to
+enable directory sharing.
+
+After you grant permissions, you will see a new directory in File Explorer on
+the remote desktop. The directory will have the same name as the directory on
+your local machine, but with "on teleport" appended to the name. 
```

Comment:
I think this is worthy of a screenshot below here of the File Explorer with the shared directory highlighted in the side bar.

<img width="1388" alt="image" src="https://user-images.githubusercontent.com/13578537/185966706-3ceb05b4-6671-4e26-872a-96214e8f9a01.png">

Reason being, I'm imagining that it might be slightly confusing for a user when they first share a directory, grant permissions, and then nothing much happens in the UI if they don't already have File Explorer open.